### DISCUSSION TITLE
Add A Video Tutorial

### DISCUSSION THREAD

**Discussion by BrutusBondBTC** [MutinyWallet/mutiny-node#1181] [Ideas]
I made a Mutiny wallet Tutorial [video on flarepub](https://flare.pub/w/naddr1qqrhx4r5dckhwaspzpmhxue69uhkummnw3ezuamfdejsygr4hu34xxhflxxx9x2m5pcereygat28t96nw8tr6l0ag677r0agj5psgqqqskasjdvqkf). I would like to add it to a github repository, but I'm not sure where it would go.

**Comment by AnthonyRonning** [comment]
Hey! This is great. Thank you so much. 

I made a new page on our wiki for tutorials: https://github.com/MutinyWallet/mutiny-web/wiki/Tutorials
